docs/developer-docs/latest/update-migration-guides/migration-guides/v4/plugin/migrate-back-end.md

```diff
@@ -0,0 +1,211 @@
+
+3. Define the content-type target with the `target` property.<br/>The value should be a string following the `api::api-name.content-type-name` or `plugin::plugin-name.content-type-name` syntax convention.
+
+4. (_optional_) In [bidirectional relations](/developer-docs/latest/development/backend-customization/models.md#relations), define `mappedBy` and `inversedBy` properties on each content-type.
+
+::: details Example of all possible relations between an article and an author content-types
+
+  ```json
+  // path: ./src/plugins/my-plugin/server/content-types/article/schema.json
+  
+  // Attributes for the Article content-type
+  "articleHasOneAuthor": {
+    "type": "relation",
+    "relation": "oneToOne",
+    "target": "api::author.author"
+  },
+  "articleHasAndBelongsToOneAuthor": {
+    "type": "relation",
+    "relation": "oneToOne",
+    "target": "api::author.author",
+    "inversedBy": "article"
+  },
+  "articleBelongsToManyAuthors": {
+    "type": "relation",
+    "relation": "oneToMany",
+    "target": "api::author.author",
+    "mappedBy": "article"
+  },
+  "authorHasManyArticles": {
+    "type": "relation",
+    "relation": "manyToOne",
+    "target": "api::author.author",
+    "inversedBy": "articles"
+  },
+  "articlesHasAndBelongsToManyAuthors": {
+    "type": "relation",
+    "relation": "manyToMany",
+    "target": "api::author.author",
+    "inversedBy": "articles"
+  },
+  "articleHasManyAuthors": {
+    "type": "relation",
+    "relation": "oneToMany",
+    "target": "api::author.author"
+  }
+  ```
+
+  ```json
+  // path: ./src/plugins/my-plugin/server/content-types/author/schema.json
+
+  // Attributes for the Author content-type
+  "article": {
+    "type": "relation",
+    "relation": "manyToOne",
+    "target": "api::article.article",
+    "inversedBy": "articleBelongsToManyAuthors"
+  },
+  "articles": {
+    "type": "relation",
+    "relation": "manyToMany",
+    "target": "api::article.article",
+    "inversedBy": "articlesHasAndBelongsToManyAuthors"
+  }
+  ```
+
+:::
+
+## Update plugin configuration
+
+:::callout
+Strapi v3 defines plugin configurations in a `config` folder.
+
+In Strapi v4, the default configuration of a plugin is defined as an object in the `config` property found in the `strapi-server.js` entry file (see [default plugin configuration documentation](/developer-docs/latest/developer-resources/plugin-api-reference/server.md#configuration)).
+:::
+
+To handle default plugin configurations in Strapi v4:
+
+1. In the `strapi-server.js` file, create a `config` object.
+
+2. Within the `config` object:
+   - Define a `default` key that takes an object to store the default configuration.
+   - Add a `validator` key, which is a function taking the `config` as an argument.
+
+::: details Example of a default plugin configuration
+
+  ```jsx
+  // path: ./src/plugins/my-plugin/strapi-server.js
+
+  module.exports = () => {
+  // … bootstrap, routes, controllers, etc.
+  config: {
```

Comment:
Same here, our CLI doesn't generate it like this, shouldn't the config either be a file in server like `./src/plugins/my-plugin/server/config.js` or nested under a folder like `./src/plugins/my-plugin/server/config/index.js` and pulled into the `./src/plugins/my-plugin/server/index.js`

Reply:
(see previous comments about the topic — I'm open to discuss it today with Mark and you, though, as long as we can find a solution and have the documentation consistent before tomorrow's release)

Reply:
Updated in [this commit](https://github.com/strapi/documentation/pull/663/commits/49b142e4ebfb69930b4e94596f19e8d0bfc7dbc7)